src/FixedSizeArrays.jl

```diff
@@ -85,8 +87,9 @@ axes_are_one_based(axes) = all(isone ∘ first, axes)
 
 function FixedSizeArray{T,N}(src::AbstractArray{S,N}) where {T,N,S}
     axs = axes(src)
-    axes_are_one_based(axs) ||
+    if !axes_are_one_based(axs)
         throw(DimensionMismatch("source array has a non-one-based indexing axis"))
```

Comment:
This line isn't exercised at the moment, but can probably be done in a separate PR

Reply:
xref #32